db/migrate/20190908234054_migrate_post_edit_time_limit.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class MigratePostEditTimeLimit < ActiveRecord::Migration[5.2]
+  def up
+    execute <<~SQL
+    INSERT INTO site_settings (
+      name,
+      value,
+      data_type,
+      created_at,
+      updated_at
+    )
+      SELECT
+        'tl2_post_edit_time_limit',
+        value,
+        data_type,
+        CURRENT_TIMESTAMP,
+        CURRENT_TIMESTAMP
+      FROM site_settings
+      WHERE
+        name = 'post_edit_time_limit'
+      AND
+        CAST(value AS INTEGER) > 43200
```

Comment:
I don't think we need this cast, we can just insert unconditionally, that way post upgrade everything will work exactly as it used to if you did any overriding.